packages/cli/src/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { AxeResults } from 'axe-core';
+
+export interface EventParams {
+  silentMode: boolean;
+  timer: boolean;
+  cliReporter: (...args: any[]) => void;
```

Comment:
we should type this, it is our own method...

Reply:
I would normally agree but this method is similar to a `console.log` so I think in this instance a type of `any[]` would be good for this

Reply:
This method is used similarly to a `console.log()` and I think type `any[]` is ok in this instance.